From 8ee33d230d0bdcefe37aa2e02f7b9c3aea47bc6a Mon Sep 17 00:00:00 2001
From: Sam Hartman <hartmans@debian.org>
Date: Tue, 29 Nov 2011 19:40:23 -0500
Subject: =?UTF-8?q?ticket:=20new=0Asubject:=20Respect=20locale=20for=20time=20?=
 =?UTF-8?q?display?=

Call setlocale(LC_TIME) in klist and kadmin to get locale-specific
time display.

Patch-Name: time_locale
---
 src/clients/klist/klist.c   |    1 +
 src/kadmin/cli/ss_wrapper.c |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/clients/klist/klist.c b/src/clients/klist/klist.c
index a9a9171..f0d55e8 100644
--- a/src/clients/klist/klist.c
+++ b/src/clients/klist/klist.c
@@ -119,6 +119,7 @@ main(argc, argv)
     int mode;
 
     setlocale(LC_MESSAGES, "");
+    setlocale(LC_TIME, "");
     progname = GET_PROGNAME(argv[0]);
 
     name = NULL;
diff --git a/src/kadmin/cli/ss_wrapper.c b/src/kadmin/cli/ss_wrapper.c
index f86c215..2608d8c 100644
--- a/src/kadmin/cli/ss_wrapper.c
+++ b/src/kadmin/cli/ss_wrapper.c
@@ -40,6 +40,7 @@ main(int argc, char *argv[])
     int sci_idx, code = 0;
 
     setlocale(LC_MESSAGES, "");
+    setlocale(LC_TIME, "");
     whoami = ((whoami = strrchr(argv[0], '/')) ? whoami+1 : argv[0]);
 
     request = kadmin_startup(argc, argv);
